### PR TITLE
[ADEN-10468] BingeBot: sponsored logo ad slot

### PR DESCRIPTION
**Pull request by Loczi94**

### CODE REVIEW

File: platforms/bingebot/ad-context.ts [hunk 1]
```diff
@@ -8,7 +8,16 @@ export const basicContext = {
...
+	state: {
+		adStack: [],
+	},
 	targeting: {
 		skin: 'bingebot',
 	},
+	services: {
+		instantConfig: {
+			endpoint: 'https://services.wikia.com/icbm/api/config?app=bingebot',
+			fallbackConfigKey: 'fallbackInstantConfig',
```
Comment (Bielik20) [inline platforms/bingebot/ad-context.ts:20]: We don't do that anymore. We have fallback inside context.

File: platforms/bingebot/setup/dynamic-slots/bingebot-dynamic-slots.setup.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+export const adSlotInjected = globalAction(
+	'[BingeBot] ad slot injected',
+	props<AdSlotInjectedProps>(),
+);
+
+export const destroyAdSlot = globalAction(
+	'[BingeBot] destroy ad slot',
+	props<AdSlotInjectedProps>(),
+);
```
Comment (Bielik20) [inline platforms/bingebot/setup/dynamic-slots/bingebot-dynamic-slots.setup.ts:18]: Do we need to export that?

File: platforms/bingebot/setup/dynamic-slots/bingebot-dynamic-slots.setup.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+@Injectable()
+export class BingeBotDynamicSlotsSetup implements DynamicSlotsSetup {
+	execute(): void {
+		communicationService.action$.pipe(ofType(adSlotInjected)).subscribe((action) => {
+			this.setAdStack(action.slotId);
+		});
+
+		communicationService.action$.pipe(ofType(destroyAdSlot)).subscribe((action) => {
+			const adSlot = slotService.slots[action.slotId];
```
Comment (Bielik20) [inline platforms/bingebot/setup/dynamic-slots/bingebot-dynamic-slots.setup.ts:28]: What about `slotService.get`?

File: platforms/bingebot/styles.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#sponsored_logo::before {
+	content: 'promoted by';
+	font-size: 8px;
+	font-weight: bold;
+	text-transform: uppercase;
+	margin-bottom: 5px;
+}
```
Comment (Bielik20) [inline platforms/bingebot/styles.scss:7]: How about moving it to AdEngine?

File: platforms/bingebot/index.ts [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import { Container } from '@wikia/dependency-injection';
 import { BingeBotPlatform } from './bingebot-platform';
+import './styles.scss';
```
Comment (Bielik20) [inline platforms/bingebot/index.ts:3]: ditto

**Review by Bielik20 — CHANGES_REQUESTED** [review]

File: platforms/bingebot/styles.scss (hunk 4 above)
Comment (Loczi94) [reply platforms/bingebot/styles.scss:7]: It is in AdEngine. 🙃  You mean moving it to `src/ad-products/styles/styles.scss`? My point of view is that this style applies to element which exists only in BingeBot app so imo there is no point in moving it from here.

**Review by Bielik20 — APPROVED** [review]

**Pull request merged**